bson/src/main/org/bson/BsonJavaScriptWithScope.java

```diff
@@ -105,5 +101,9 @@ public String toString() {
                + "scope=" + scope
                + '}';
     }
+
+    static BsonJavaScriptWithScope clone(final BsonJavaScriptWithScope from) {
```

Comment:
This could be cloneable since it's not immutable, but I don't see a reason anyone would want to clone it independently of a BsonDocument, so leaving it out for now.